app/controllers/concerns/training_plan_concern.rb

```diff
@@ -0,0 +1,24 @@
+module TrainingPlanConcern
+  extend ActiveSupport::Concern
+
+  def name(training)
+    name = Client.find_by(id: training.client_id).first_name + ' ' + Client.find_by(id: training.client_id).second_name
+    name
+  end
+
+  def sets(training, current_user)
+    sets = Kit.where(training_id: training.id, user_id: current_user.id).map do |kit|
```

Comment:
it's hard to understand what going on here

Reply:
Getting all kits, that belongs to current training and current user